orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
"Hypothesis creates saved models. This will run inside your testing transaction when using the test runner, but if you use the dev console this will leave debris in your database."  That's potentially going to break tests that use queryset retrieval methods and assert the results.

Looking at these docs doesn't exactly fill me with confidence.  There are a lot of ifs, buts, caveats, and limitations.  I seem to recall this being among the reasons we didn't start using it when it first came to our attention - the project just doesn't seem ready or mature enough, at least in its Django support.